jolt-core/src/jolt/trace/rv.rs

```diff
@@ -686,9 +686,11 @@ impl JoltProvableTrace for RVTraceRow {
       RV32IM::BGE  => vec![BGEInstruction(self.rs1_val.unwrap(), self.rs2_val.unwrap()).into()],
       RV32IM::BGEU => vec![BGEUInstruction(self.rs1_val.unwrap(), self.rs2_val.unwrap()).into()],
 
-      RV32IM::JAL  => vec![JALInstruction(self.pc, self.imm_u64()).into()],
-      RV32IM::JALR => vec![JALRInstruction(self.rs1_val.unwrap(), self.imm_u64()).into()],
+    //   RV32IM::JAL  => vec![JALInstruction(self.pc, self.imm_u64()).into()],
+    //   RV32IM::JALR => vec![JALRInstruction(self.rs1_val.unwrap(), self.imm_u64()).into()],
 
+      RV32IM::JAL  => vec![ADDInstruction::<32>(self.pc, self.imm_u64()).into()],
```

Comment:
Why did we swap these out for ADD?

Reply:
Turns out I misunderstood the semantics on the Jumps. It's sufficient to just use ADDs there. I think it's safe to remove the JAL and JALR instructions, but we could wait till later just to make sure.